root.spec

```diff
@@ -22,6 +22,21 @@ Requires: freetype
 
 %define keep_archives true
 
+%define subpackageDebug true
+
+%if "%{?subpackageDebug:set}" == "set"
+%define envOptions USER_CXXFLAGS="-fdebug-prefix-map=%{cmsroot}=%{installroot} -fdebug-prefix-map=%{instroot}=%{installroot} -g -gdwarf-2 -O0"
+%endif
+# For non-linux archs disable subpackageDebug otherwise add dwz.file build dependencies.
+%if "%{?subpackageDebug:set}" == "set"
+%if %islinux
+BuildRequires: dwz file
```

Comment:
Why do you need this if you are not optimising debug information? You might want to optimise it, otherwise debug information could be huge for ROOT + LLVM.